### ISSUE TITLE
ExchangeType should be from str type

### ISSUE THREAD

**Issue by liortct** [pika/pika#1409]
I am trying to use `ExhangeType` but PEP8 give me a warning.
PEP8 give me the following warning `Expected type 'str', got 'ExchangeType' instead`. 
It's because `ExhangeType` enum is inheritance only from enum and not from str as needed.

Versions:
pika=1.3.1
Python=3.8.10

Example code:
```
import pika
from pika.exchange_type import ExchangeType

connection = pika.BlockingConnection(pika.ConnectionParameters('localhost'))
channel = connection.channel()

channel.exchange_declare(exchange='logs', exchange_type=ExchangeType.topic)
```

**Comment by lukebakken** [comment]
How did you get the PEP8 warning? Did you run a program like pylint?

**Comment by liortct** [comment]
I got the warning simply via the PyCharm.
I will check about pylint and mypy.

**Comment by lukebakken** [comment]
It is just a warning and can be ignored. If you have a fix a pull request would be appreciated.

My guess is that `ExchangeType` should be a subclass of [`StrEnum`](https://docs.python.org/3/library/enum.html#enum.StrEnum)

**Comment by lukebakken** [comment]
Fixed by #1410
